_docs/aws_resources.md

```diff
@@ -35,7 +35,9 @@ permalink: /aws_resources.html
 When considering importing and exporting database information, there are a number of approaches a developer can take. Two common approaches involving creating your own function, using Lambda, or any other scripting service, or using a pipeline. This section covers some of the different use cases for those options in AWS, as well as provides some sample code for exporting/importing by hand.
```

Comment:
Lambda is not a scripting service, it is a serverless compute service. The difference is slight but important, since the term "scripting" has a specific connotation.
```suggestion
When considering importing and exporting database information, there are a number of approaches a developer can take. Two common approaches involving creating your own function using a service like AWS Lambda, or using a pipeline. This section covers some of the different use cases for those options in AWS, as well as provides some sample code for exporting/importing by hand.
```